pkg/roachpb/api.go

```diff
@@ -966,3 +966,23 @@ func (b *BulkOpSummary) Add(other BulkOpSummary) {
 	b.IndexEntries += other.IndexEntries
 	b.SystemRecords += other.SystemRecords
 }
+
+// Equal method is just used to verify whether two ComputeChecksumRequest is equal.
+func (ccr *ComputeChecksumRequest) Equal(cr *ComputeChecksumRequest) bool {
```

Comment:
OK, seems like you could change the existing implementation to `TypeSafeEqual` and then generate this less-safe `Equal`.